### PR TITLE
Update with new dbt web version

### PR DESCRIPTION
**Pull request by rlh1994**

### CODE REVIEW

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ Redshift and Postgres use a [shredded](/docs/pipeline-components-and-application
 | `yauaa_context`       | `nl_basjes_yauaa_context_1`                            |
 | `consent_cmp_visible` | `com_snowplowanalytics_snowplow_cmp_visible_1`         |
 | `consent_preferences` | `com_snowplowanalytics_snowplow_consent_preferences_1` |
+| `cwv_context`         | `com_snowplowanalytics_snowplow_web_vitals_1`          |
```
Comment (agnessnowplow) [inline docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:113]: these are events

**Review by agnessnowplow — APPROVED** [review]

File: docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md (hunk 1 above)
Comment (agnessnowplow) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:113]: FYI as it is only changed in unified

Comment (rlh1994) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:113]: Yeah it's not ideal but didn't want to make a breaking change, that can be a v1 problem :D

Comment (agnessnowplow) [reply docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md:113]: sure

**Pull request merged**